From a0287334c8038de8ad3b4b0492f2d5b809f69037 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Fri, 17 Aug 2012 10:09:35 +0300
Subject: [PATCH 289/370] conf: c2_conf_reg_init() should return `void'

The only function being called from c2_conf_reg_init() returns `void'.
There is no point in return value which is always 0.

LogD #750
---
 conf/reg.c | 2 +-
 conf/reg.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/conf/reg.c b/conf/reg.c
index 99dacef..bc10e5a 100644
--- a/conf/reg.c
+++ b/conf/reg.c
@@ -59,7 +59,7 @@ C2_TL_DESCR_DEFINE(reg, "registered c2_conf_obj-s", static, struct objptr,
 		   p_linkage, p_magic, OBJPTR_MAGIC, CONF_REG_MAGIC);
 C2_TL_DEFINE(reg, static, struct objptr);
 
-int c2_conf_reg_init(struct c2_conf_reg *reg)
+void c2_conf_reg_init(struct c2_conf_reg *reg)
 {
 	reg_tlist_init(&reg->r_ptrs);
 }
diff --git a/conf/reg.h b/conf/reg.h
index cc3c978..c400154 100644
--- a/conf/reg.h
+++ b/conf/reg.h
@@ -70,7 +70,7 @@ struct c2_conf_reg {
 };
 
 /** Initialises a registry. */
-int c2_conf_reg_init(struct c2_conf_reg *reg);
+void c2_conf_reg_init(struct c2_conf_reg *reg);
 
 /**
  * Finalises a registry, freeing all the registered configuration objects.
-- 
1.8.3.2

